### PR TITLE
Add feature to GetList

### PR DESCRIPTION
**Pull request by alexanderwanyoike**
## Overview 

Add support for `additional_information_json` on the get list plugin for order repository.

## Tasks
- [x] Update plugin
- [ ] Test

### CODE REVIEW

File: Plugin/OrderRepositoryPlugin.php [hunk 1]
```diff
@@ -20,11 +21,26 @@ public function __construct(OrderPaymentExtensionFactory $orderPaymentExtensionF
...
     public function afterGet(OrderRepositoryInterface $orderRepository, OrderInterface $order)
+    {
+        $this->changeAdditionalInformation($order);
+        return $order;
+    }
+
+    public function afterGetList(OrderRepositoryInterface $orderRepository, OrderSearchResultInterface $orders)
+    {
+        foreach ($orders->getItems() as $key => $order)
+        {
```
Comment (BillTsarnas) [inline Plugin/OrderRepositoryPlugin.php]: move the opening bracket up :)

File: Plugin/OrderRepositoryPlugin.php [hunk 2]
```diff
@@ -20,11 +21,26 @@ public function __construct(OrderPaymentExtensionFactory $orderPaymentExtensionF
      * and add the additional information as a json encoded string to an extension attribute.
      */
     public function afterGet(OrderRepositoryInterface $orderRepository, OrderInterface $order)
+    {
+        $this->changeAdditionalInformation($order);
+        return $order;
+    }
+
+    public function afterGetList(OrderRepositoryInterface $orderRepository, OrderSearchResultInterface $orders)
```
Comment (BillTsarnas) [inline Plugin/OrderRepositoryPlugin.php]: this is a minor detail, `$orderRepository ` can be renamed to `$subject` - we follow this convention when writing plugin methods

File: Plugin/OrderRepositoryPlugin.php [hunk 3]
```diff
@@ -35,8 +51,5 @@ public function afterGet(OrderRepositoryInterface $orderRepository, OrderInterfa
         if (method_exists($payment, 'unsAdditionalInformation')) {
             $payment->unsAdditionalInformation();
         }
-
-        return $order;
-
     }
 }
```
Comment (BillTsarnas) [inline Plugin/OrderRepositoryPlugin.php]: newline here

File: Plugin/OrderRepositoryPlugin.php [hunk 4]
```diff
@@ -20,11 +21,26 @@ public function __construct(OrderPaymentExtensionFactory $orderPaymentExtensionF
...
+    {
+        foreach ($orders->getItems() as $key => $order)
+        {
+            $this->changeAdditionalInformation($order);
+        }
+
+        return $orders;
+    }
+
+    private function changeAdditionalInformation(OrderInterface &$order): OrderInterface
```
Comment (BillTsarnas) [inline Plugin/OrderRepositoryPlugin.php]: we should avoid calling stuff by reference (changing and returning an `OrderInterface` item will do finely)

Comment (alexanderwanyoike) [reply Plugin/OrderRepositoryPlugin.php]: Even if its in a set?

**Pull request merged**